data/morphology/grab_unimorph_data.py

```diff
@@ -1,48 +1,49 @@
 #!/usr/bin/env python
 """Downloads UniMorph morphological paradigms data."""
-
 import json
 import logging
 import os
-import requests
+from typing import Dict
 import time
 
-from typing import Dict
+
+#import requests in its own import block
+import requests
 
 
 UNIMORPH_DICT_PATH = "unimorph_languages.json"
 
 
-def download(data_to_grab: Dict[str, str]) -> Dict[str, str]:
+def download():
     to_retry = {}
-    os.mkdir("tsv")
-    for language, url in data_to_grab.items():
-        with requests.get(url, stream=True) as response:
-            logging.info("Downloading: %s", language)
+    os.makedirs("tsv")
+    with open(UNIMORPH_DICT_PATH) as jfile:
+        data_to_grab = json.load(jfile)
+    for k, v in data_to_grab.items():
+        with requests.get(data_to_grab[k], stream=True) as response:
+            target_path = data_to_grab[k].split("/")[-1]
+            logging.info("Downloading: %s", target_path)
             if response.status_code == 200:
-                with open(f"tsv/{language}.tsv", "wb") as f:
-                    f.write(response.content)
+                with open(f"unimorph_for_split/{target_path}.tsv", "w+") as sink:
+                    print(response.text,file=sink)
             else:
                 logging.info(
-                    "Status code %d while downloading %s",
-                    response.status_code,
-                    language,
+                "Status code %s while downloading %s",
+                response.status_code,
+                target_path
                 )
-                to_retry[language] = data_to_grab[language]
-        # 30 seconds appears to not be enough, 60-70 seconds works well
-        # but takes a long time.
-        time.sleep(45)
-    return to_retry
-
+            time.sleep(45)
 
-def main() -> None:
-    with open(UNIMORPH_DICT_PATH, "r", encoding="utf-8") as langs:
-        languages = json.load(langs)
+       
+    def main() -> None:
+        with open(WORTSCHATZ_DICT_PATH, "r", encoding="utf-8") as langs:
+            languages = json.load(langs)
     # Hack for repeatedly attempting to download Wortschatz data
```

Comment:
A line comment should be as indented as the code it refers to. So here it should line up wtih `langs_to_retry`.